R/default_split_funs.R

```diff
@@ -73,9 +73,10 @@ keep_split_levels <- function(only, reorder = TRUE) {
     unique_vals <- .get_unique_levels(varvec)
 
     # Error in case not all levels are present
-    if (!all(only %in% unique_vals)) {
+    if (length(varvec) > 0 && !all(only %in% unique_vals)) {
```

Comment:
If we want only to check for factors we can readd `is.factor(varvec)` instead of the above change. I much prefer to check also if the vector is a character if the selected values are present.

Reply:
I agree we should account for character vectors as well.

Reply:
I think it is settled then! Thanks Emily :)